doc/cookbook/compression_threads.cpp

```diff
@@ -2,17 +2,17 @@
 //![example]
 #include <seqan3/io/all.hpp>
 
-
-// The `bgzf_thread_count` is a variable that can only be changed during run time of the program.
-// This does not work, the value must be overwritten within a function.
-// seqan3::contrib::bgzf_thread_count = 1u; // Doesn't work
+// The `bgzf_thread_count` is a variable that can only be changed during the runtime of a program.
+// The following does not work, the value must be overwritten within a function:
+// seqan3::contrib::bgzf_thread_count = 1u; // Does not work.
 
 int main()
 {
-    // Here we change the number of threads to `1`. This is a global
-    // change and will effect the every future bgzf decompression call
-    // which doesn't specify an explicit thread count parameter.
-    // Already running decompression calls are unaffected by this
+    // Here, we change the number of threads to `1`.
+    // This is a global change and will affect every future bgzf (de-)compression.
+    // However, running (de-)compressions will not be affected.
```

Comment:
```suggestion
    // However, already running (de-)compressions will not be affected.
```

Reply:
What's the difference between `running` and `already running`?
Present participle already indicates that it's currently running.